### ISSUE TITLE
May I ask for a verification test?

### ISSUE THREAD

**Issue by fenixlin** [hanxiao/bert-as-service#83]
Hi,
I successfully deployed the server, however it just utilized around 700MB GPU memory, which make me doubt if something have gone wrong. (In https://github.com/google-research/bert, it suggested 12GB GPU memory at minimum, and no specification are shown from your README.md)
I tried comparing the result generated by GPU and CPU and they are nearly the same.
Could you offer a test code to check whether the server is giving correct result vector if you have time (using default parameter and standard model is ok. better if you can offer chinese model's) ? Verifying the head of a word's result vector will do as the simplest case. It really helps!

Environment I'm using:
Tesla K80
tensorflow 1.10.0
GPUtil 1.3.0
pyzmq 17.1.2

Thank you very much!

**Comment by hanxiao** [comment]
Easy, 700MB GPU memory is normal. Considering a 12-layer, 768-hidden, 12-heads BERT has 110 million parameters in `tf.float32`, which takes 110 million * 32 bits = 400mb for the feedforward path (i.e no gradient and auxiliary variables).

But you are right, there should be some test to ensure that we are not accidentally break the model.

**Comment by hanxiao** [comment]
Starting from 1.5.5, `bert-as-service` freeze and optimize the graph before serving to reduce the memory footprint. The GPU memory consumption of `bert-as-service` should be constant regardless pretrained/fine-tuned model, and it is even smaller when you use deeper `pooling_layer`. Please do 
```bash
pip install -U bert-serving-server bert-serving-client
```
for the upgrade.
